### PR TITLE
[android] added FrostWire splash screen

### PR DESCRIPTION
**Pull request by marcelinkaaa**

### CODE REVIEW

**Comment by aldenml** [comment]
sounds good the new sizes, let work out the details now

**Comment by aldenml** [comment]
- Can you remove `android/res/drawable-nodpi/frostwire_splash.9.png`?
- If you keep the name `"Theme.FrostWire"` in the style, you can refer to it like `R.style.Theme_FrostWire` in the source code (note the `_`)

**Comment by marcelinkaaa** [comment]
@aldenml removed. Also didn't know that '_' will work in this situation. Thanks.

File: android/res/drawable/frostwire_splash_screen.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (aldenml) [inline android/res/drawable/frostwire_splash_screen.xml]: note: missing header

**Pull request merged**